core-web/src/main/java/org/silverpeas/core/web/util/viewgenerator/html/progressmessage/AbstractProgressMessage.java

```diff
@@ -39,6 +39,7 @@
 import org.silverpeas.core.util.LocalizationBundle;
 import org.apache.ecs.ElementContainer;
 import org.apache.ecs.xhtml.script;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.apache.ecs.xhtml.script'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)